admin/sign-in.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useContext } from "react";
+import { AuthContext } from "./authContext";
+import "./sign-in.scss";
+
+export const SignIn = () => {
+  const { handleSignIn } = useContext(AuthContext);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e: any) => {
+    e.preventDefault();
+    handleSignIn(email, password);
+  };
+
+  return (
+    <form className="sign-in" onSubmit={e => handleSubmit(e)}>
+      <label>
+        E-mail
+        <input type="text" onChange={e => setEmail(e.target.value)}></input>
+      </label>
+      <label>
+        Password
+        <input type="text" onChange={e => setPassword(e.target.value)}></input>
```

Comment:
her vil du sikkert bruke `<input type="password" />`, for å skjule passordet underveis

Reply:
Tja, det er bare jeg som skal bruke siden. Er ikke så redd for at folk skal se passordet :)

Reply:
For all del, men det er best practice når det kommer til passord (og du kan bl.a. ikke kopiere ting ut ifra feltet), og dette prosjektet handler jo tross alt om å naile best practices i webutvikling ;-)